bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/it/CloudBigtableBeamITTest.java

```diff
@@ -117,21 +120,24 @@
   private static final String adminEndpoint =
       System.getProperty(BIGTABLE_ADMIN_HOST_KEY, BIGTABLE_ADMIN_HOST_DEFAULT);
   private static final String TABLE_NAME_STR =
-      System.getProperty("tableName", "BeamCloudBigtableIOIntegrationTest");
+      System.getProperty("google.dataflow.tableName", "BeamCloudBigtableIOIntegrationTest");
 
   private static final TableName TABLE_NAME = TableName.valueOf(TABLE_NAME_STR);
-  private static final byte[] FAMILY = Bytes.toBytes("test-family");
+  private static final byte[] FAMILY = Bytes.toBytes("google.dataflow.test-family");
```

Comment:
why does the table schema need to be configurable?

Reply:
It's still hardcoded, changed by mistake. I will restore the original one value.